src/commands/authorize/authorizeCommand.ts

```diff
@@ -0,0 +1,61 @@
+interface AuthorizeInterface {
+    state: string;
+    client_id: string;
+    redirect_uri: string;
+    scope: string;
+    cookie?: string | undefined;
```

Comment:
undefined is inferred by virtue of using an optional type so we can remove the `| undefined`

Reply:
Yes, of course.